lib/relay.js

```diff
@@ -186,6 +187,10 @@ function responseHandler(filterRules, config, io) {
     { url, headers = {}, method, body = null, streamingID = '' } = {},
     emit,
   ) => {
+    if (url && url.startsWith(relayGit.GIT_CLIENT_REDIRECT_PATH)) {
+      return relayGit.response(config, io)(brokerToken)({ url, headers, method, body, streamingID }, emit);
```

Comment:
Question: just wondering: since `relayGit.response` is used only here and called directly, why did you go for a curried function call? Probably reasonable to simplify future refactoring between `relay.js`/`relay-git.js`?